### DISCUSSION TITLE
2024-06-27

### DISCUSSION THREAD

**Discussion by jeasonstudio** [ant-design/ant-design-web3#1005] [中文社区双周会]
上次周会记录：https://github.com/ant-design/ant-design-web3/discussions/947

**Comment by jeasonstudio** [comment]
支持 TON 连接：https://github.com/ant-design/ant-design-web3/issues/1008

**Comment by jeasonstudio** [comment]
PR 跟进：
* https://github.com/ant-design/ant-design-web3/pull/1003
* https://github.com/ant-design/ant-design-web3/pull/988
* https://github.com/ant-design/ant-design-web3/pull/991

**Comment by gin-lsl** [comment]
要不要在 TokenSelect 中添加一个链多选形式：
![image](https://github.com/ant-design/ant-design-web3/assets/18319675/1c289d90-3772-4a54-bcd4-4abeacb23a4f)

**Reply by jeasonstudio** [reply]
需要wenqing一起看下，是沿用 antd 的多选样式还是用 zan 的样式

**Reply by MarioJames** [reply]
和文卿确认了下 说是多选时最多展示三个选中 Token 的 Icon，超过需要点开下拉查看
